lib/Yapsi.pm

```diff
@@ -185,7 +186,7 @@ class Yapsi::Perl6::Actions {
         my $block = @blockstack.pop;
         my $name = $block.name;
         $block.vars = @vars.list;
-        $block.children.push($<statementlist><statement>».ast.grep(*.defined));
+        $block.children.push: | $<statementlist><statement>».ast.grep(*.defined);
```

Comment:
A more idiomatic change would be to switch from `push` to `append` here, instead of creating a Slip